resources/gui/css/sidebar.css

```diff
@@ -1,27 +1,28 @@
-.side-bar {
+#sidebar {
     width: 450px;
-    position: absolute;
-    height: 100%;
+    height: 100vh;
+    margin-top: -25px;
     overflow-x: hidden;
```

Comment:
Looks very nice! You could add `padding-bottom: 100px;` here in order to allow scrolling to the very end of the sidebar.

Reply:
Padded things up!